fuse_doc/doc/getting_started.rst

```diff
@@ -0,0 +1,253 @@
+.. _getting_started:
+
+Tutorial: Getting Started
+#########################
+
+In this tutorial, we will show users how to configure `fuse` to combine sensor data from multiple sensors and produce a state estimate.
+
+To start, you'll need a launch file and a yaml configuration file. We'll create both:
+
+.. code-block:: bash
+
+  mkdir ~/fuse_tutorials
+  cd ~/fuse_tutorials
+  touch fuse_simple_tutorial.launch
+  touch fuse_simple_tutorial.yaml
+
+Launch File
+***********
+
+The core state estimation node in fuse is known as the `fixed_lag_smoother_node`. We'll begin by adding it to our `fuse_simple_tutorial.launch` file:
+
+.. code-block:: xml
+
+  <launch>
+    <param name="/use_sim_time" value="true"/>
+
+    <node name="state_estimator" pkg="fuse_optimizers" type="fixed_lag_smoother_node">
+      <rosparam command="load" file="$(env HOME)/fuse_tutorials/fuse_simple_tutorial.yaml"/>
+    </node>
+  </launch>
+
+Basic Configuration
+*******************
+
+Our initial configuration will simply fuse a single wheel odometry sensor. Note that this example will not produce any output. We'll cover that later in the tutorial. Open up `fuse_simple_tutorial.yaml` and paste this into it:
+
+.. code-block:: yaml
+
+  # Fixed-lag smoother configuration
+  optimization_frequency: 20
+  transaction_timeout: 0.01
+  lag_duration: 0.5
+
+  solver_options:
+    max_num_iterations: 5
+
+  motion_models:
+    unicycle_motion_model:
+      type: fuse_models::Unicycle2D
+
+  unicycle_motion_model:
+    #                         x      y      yaw    vx     vy     vyaw   ax   ay
+    process_noise_diagonal: [0.100, 0.100, 0.100, 0.100, 0.100, 0.100, 0.1, 0.1]
+    scale_process_noise: true
```

Comment:
TBH I've never got good results using `scale_process_noise`. It's probably fine for this tutorial, but it could also be skipped.

If the state velocity estimate is zero, it could lead to a zero pose covariance after the process noise is scaled by the state velocity estimate norm. If set, I believe `velocity_norm_min` should be set as well to avoid this potential issue.

Reply:
I'll try removing it and test it against the bag file I have.